code/_onclick/hud/rendering/plane_masters/plane_master_subtypes.dm

```diff
@@ -584,6 +584,18 @@
 	name = "Examine Balloons"
 	documentation = "The balloons that appear above objects (often wallmounts) when holding shift."
 	plane = EXAMINE_BALLOONS_PLANE
+	invisibility = INVISIBILITY_ABSTRACT
```

Comment:
there should be no reason to do this. instead of setting invisibiity just remove/readd the relay to the GAME_WORLD plate